### ISSUE TITLE
Vertical alignment using Revealjs

### ISSUE THREAD

**Issue by mroavi** [quarto-dev/quarto-cli#923]
I noticed that by default `h1` slides are vertically centered while `h2` slides are top-aligned.
How can I make `h2` slides vertically centered (like `h1` slides)? I've spent hours trying to tweak this but haven't found out how.

I found that this is how you would achieve this using Revealjs's markdown:

```md
<section data-markdown>
    <script type="text/template">
    <!-- .slide: class="center" -->
        Markdown content
    </script>
</section>
```

I also found that you can control text-alignment horizontally with the scss theme, e.g.:

```scss
$presentation-slide-text-align: left !default;
```

Thanks in advance!

**Comment by jjallaire** [comment]
You should be able to just specify the `center: true` option and it will center the H2s. LMK if this doesn't work.

**Comment by mroavi** [comment]
Doesn't work for me. Here is how I tested it:

```markdown
---
title: "Habits"
author: "John Doe"
format:
  revealjs:
    center: true
---

## H2
- Bullet point
- Bullet point
```

**Comment by jjallaire** [comment]
Yikes! That's surprising. I believe I've got a fix for this here: https://github.com/quarto-dev/quarto-cli/commit/035cb0916a6a177ce62bcceca6fb6bc804e2c4ec

Will be available in the next build (v0.9.425+)
